packages/sources/tiingo/src/endpoint/forex.ts

```diff
@@ -37,15 +38,38 @@ interface ResponseSchema {
   midPrice: number
 }
 
+export type TOptions = {
+  url: string
+  inverse?: boolean
+}
+
+const getUrl = (from: string, to: string) => ({
+  url: util.buildUrlPath('/tiingo/fx/:ticker/top', { ticker: `${from}${to}`.toLowerCase() }),
+})
+
+const getIncludesOptions = (
+  //@ts-expect-error no-unused-vars
+  validator: Validator<TInputParameters>,
```

Comment:
```suggestion
  _: Validator<TInputParameters>,
```
Declaring the unused parameter as an underscore will tell the TS transpiler that it is unused.

Reply:
Could you do the same in `kaiko` since I did the same thing there originally?

Reply:
I see the same in openexchangerates too. Should I take care of both of those under this PR?

Reply:
Yes please!

Reply:
The latest commit includes the change for Kaiko and Openexchangerates